docs/reference.md

```diff
@@ -99,39 +99,36 @@ folding in MHLO, at which point we'll improve ergonomics of the code snippet
 above (e.g. we could have a helper function which packs constant operands into
 `Tensor` objects and unpacks `Tensor` results into `OpFoldResult`).
 
-## Testing the interpreter
+## Testing the StableHLO interpreter
 
 The interpreter takes as inputs (A) a StableHLO program, and (B) data values to
 be fed to the program, and generates output data values, which are matched
-against the user-provided expected data values.
-
-In the current implementation, we package the inputs (MLIR program + input data
-values) and outputs in a
-[lit-based](https://llvm.org/docs/CommandGuide/lit.html) test as follows:
+against the user-provided expected data values. The data values (B) are
+hard-coded in the program itself using `stablehlo.constant` operations. The
+interpreter evaluates the input program. The output(s) of the op under test
+is checked via assertions (e.g. `check.eq`, `check.almost_eq`), as shown below.
```

Comment:
Another use of assertions which would be good to migrate away. Replace with "checks"? (Same below in the document).